### ISSUE TITLE
[WARNING] Invalid issue syntax, ignoring: src/main/resources/xxxx/meta/main.yml:1: schema (schema[meta])

### ISSUE THREAD

**Issue by mandracke76** [ansible/ansible-lint#2394]
##### Summary

Hello,

I am developing an Ansible role for my project and when the ansible-lint check my meta file, it is set a WARNING about it.
I was check about all information about meta file but I don't understand why I have this WARNING.
Is it a bug ?

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)

[INFO] Sensor Ansible Lint Sensor [ansible]
[INFO] ansible version:
[INFO] ansible [core 2.12.8]
[INFO]   config file = None
[INFO]   configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
[INFO]   ansible python module location = /usr/local/lib/python3.9/dist-packages/ansible
[INFO]   ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
[INFO]   executable location = /usr/local/bin/ansible
[INFO]   python version = 3.9.2 (default, Feb 28 2021, 17:03:44) [GCC 10.2.1 20210110]
[INFO]   jinja version = 3.1.2
[INFO]   libyaml = True
[INFO] ansible-lint version:
[INFO] ansible-lint 6.5.0 using ansible 2.12.8
**[WARNING] Invalid issue syntax, ignoring: src/main/resources/xxxx/meta/main.yml:1: schema (schema[meta])
[WARNING] Invalid issue syntax, ignoring: src/main/resources/xxxx/meta/main.yml:1: schema (schema[meta])**
[INFO] Sensor Ansible Lint Sensor [ansible] (done) | time=19945ms

```

- ansible installation method: pip3 ( Docker )
- ansible-lint installation method: pip3  ( Docker )

```
FROM our.docker.registry/debian:11.2
LABEL maintainer="me"

RUN mkdir /root/.config
ADD ansible-lint /root/.config/ansible-lint
RUN apt-get update -y  &&  apt-get install --fix-missing
RUN apt-get install -y python3 curl gcc python3-pip sshpass

RUN pip3 install --upgrade pip
RUN pip3 install cryptography
RUN pip3 install --no-cache-dir --no-compile "ansible-lint==6.5.0" "ansible>=5.9.0,<6.00.0"

WORKDIR /data
CMD ["ansible-lint" "--version"]
```
ansible-lint:
```yaml
skip_list:
  - var-naming
  - schema
  - fqcn-builtins
```

```
FROM our.docker.registry/maven:3.6-jdk-11
LABEL maintainer="me"

RUN apt-get update && \
    apt-get install python-pip -y && \
    pip install ansible-lint

```

My meta file: src/main/resources/xxxx/meta/main.yml
```
galaxy_info:
  author: Me
  description: to deploy my role
  company: us

  # If the issue tracker for your role is not on github, uncomment the
  # next line and provide a value
  # issue_tracker_url: 

  # Some suggested licenses:
  # - BSD (default)
  # - MIT
  # - GPLv2
  # - GPLv3
  # - Apache
  # - CC-BY
  license: GPLv2

  min_ansible_version: 1.2

  # If this a Container Enabled role, provide the minimum Ansible Container version.
  # min_ansible_container_version:

  # Optionally specify the branch Galaxy will use when accessing the GitHub
  # repo for this role. During role install, if no tags are available,
  # Galaxy will use this branch. During import Galaxy will access files on
  # this branch. If Travis integration is configured, only notifications for this
  # branch will be accepted. Otherwise, in all cases, the repo's default branch
  # (usually master) will be used.
  # github_branch:

  #
  # platforms is a list of platforms, and each platform has a name and a list of versions.
  #
  platforms:
    - name: ArchLinux
      versions:
        - all
  # - name: SomePlatform
  #   versions:
  #   - all
  #   - 1.0
  #   - 7
  #   - 99.99

  galaxy_tags: []
    # List tags for your role here, one per line. A tag is a keyword that describes
    # and categorizes the role. Users find roles by searching for tags. Be sure to
    # remove the '[]' above, if you add tags to this list.
    #
    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
    #       Maximum 20 tags per role.

dependencies: []
  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
  # if you add dependencies to this list.

```
##### OS / ENVIRONMENT

GitLab CI with docker runner: debian:11.2

[INFO] SonarQube version: 8.9.7
[INFO] Default locale: "en", source code encoding: "UTF-8"

Execution mode: 
```bash
mvn ${MAVEN_CLI_OPTS} compile org.sonarsource.scanner.maven:sonar-maven-plugin:3.9.0.2155:sonar -Pgitlabci -Duser.home=. -Dsonar.projectKey=${SONAR_PROJECT_KEY/\//_} -Dsonar.gitlab.project_id=${CI_PROJECT_PATH} -Dsonar.gitlab.commit_sha=${CI_COMMIT_SHA} -Dsonar.gitlab.ref_name=${CI_COMMIT_REF_NAME} "${SONAR_PROJECT_NAME}"
```
